docs/mdbook/src/android/map-options-guide.md

```diff
@@ -0,0 +1,308 @@
+
+
+Common configurations can be set:
+- Map base uri
+- Camera
+- Zoom
+- Pitch
+- Gestures
+- Compass
+- Logo
+- Attribution
+- Placement of above elements on the Map and more
+
+We will see how to achieve these configurations in XML layout and programmatically in Activity code step by step
+
+1. Creating MapView namespace and provide some data in layout file
+  ```xml
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activity.options.MapOptionsXmlActivity">
+
+    <org.maplibre.android.maps.MapView
+        android:id="@+id/mapView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:maplibre_apiBaseUri="https://api.maplibre.org"
+        app:maplibre_cameraBearing="34.0"
+        app:maplibre_cameraPitchMax="90.0"
+        app:maplibre_cameraPitchMin="0.0"
+        app:maplibre_cameraTargetLat="52.519003"
+        app:maplibre_cameraTargetLng="13.400972"
+        app:maplibre_cameraTilt="25.0"
+        app:maplibre_cameraZoom="16"
+        app:maplibre_cameraZoomMax="34.0"
+        app:maplibre_cameraZoomMin="15.0"
+        app:maplibre_localIdeographFontFamilies="@array/array_local_ideograph_family_test"
+        app:maplibre_localIdeographFontFamily="Droid Sans"
+        app:maplibre_uiCompass="true"
+        app:maplibre_uiCompassFadeFacingNorth="true"
+        app:maplibre_uiCompassGravity="top|end"
+        app:maplibre_uiDoubleTapGestures="true"
+        app:maplibre_uiHorizontalScrollGestures="true"
+        app:maplibre_uiRotateGestures="true"
+        app:maplibre_uiScrollGestures="true"
+        app:maplibre_uiTiltGestures="true"
+        app:maplibre_uiZoomGestures="true" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
+```
+We can give any other existing values to `maplibre` tags and  only need to create Mapview and MapLibreMap objects with simple setup in Activity
+```kotlin
+class MapOptionsXmlActivity : AppCompatActivity(), OnMapReadyCallback {
+    private lateinit var mapView: MapView
+    private lateinit var maplibreMap: MapLibreMap
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_map_options_xml)
+        mapView = findViewById(R.id.mapView)
+        mapView.onCreate(savedInstanceState)
+        mapView.getMapAsync(this)
+    }
+
+    override fun onMapReady(maplibreMap: MapLibreMap) {
+        this.maplibreMap = maplibreMap
+        this.maplibreMap.setStyle(
+            Style.Builder().fromUri(TestStyles.getPredefinedStyleWithFallback("Streets"))
```

Comment:
Maybe don't use this because it is an internal class to the test app. Instead use something that users could copy paste in their own apps.

Reply:
Do you mean I need to change how to apply style here , or full code here is inappropriate?

Reply:
Yes just use a hardcoded style URL like demotiles.